WordPress/WordPressTest/PageModelTest.m

```diff
@@ -0,0 +1,62 @@
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+#import "TestContextManager.h"
+#import "Page.h"
+
+@interface PageModelTest : XCTestCase
+
+@end
+
+@implementation PageModelTest
+
+- (void)setUp {
+    [super setUp];
+}
+
+- (void)tearDown {
+    [super tearDown];
+}
+
+- (Page *)newPageForTesting
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+    Page *page = [NSEntityDescription insertNewObjectForEntityForName:NSStringFromClass([Page class])
```

Comment:
This is not necessarily bad.  But I think it's worth pointing out that you can avoid dealing with contexts for unit tests using the following options:

`Page *page = OCMClassMock([Page class]);`

or 

`Page *page = OCMStrictClassMock([Page class]);`

Feel free to ignore this comment if you still prefer to use contexts.